heareval/embeddings/task_embeddings.py

```diff
@@ -38,7 +38,7 @@
 from torch.utils.data import DataLoader, Dataset
 from tqdm.auto import tqdm
 
-import wandb
+# import wandb
```

Comment:
What's up with the wandb stuff here?

Reply:
I'm sick of auth'ing wand for every GCP run. But when you use it, you get stats charts like the following

![image](https://user-images.githubusercontent.com/65918/131315844-b6b716aa-3ff2-4abd-8beb-8b9ce9cb1f76.png)